### PR TITLE
Add JSON objects for all vanilla music

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

File: objects/rct2/music/rct2.music.fairground.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "version": "1.0",
+    "originalId": "0000008E|#MUSIC01|00000000",
+    "sourceGame": ["rct2"],
+    "objectType": "music",
+    "properties": {
+        "rideTypes": ["merry_go_round"],
+        "tracks": [
+            {
+                "source": "$RCT2:DATA/css4.dat",
+                "name": "Children of the regiment",
```
Comment (Gymnasiast) [inline objects/rct2/music/rct2.music.fairground.json:13]: This field should list the actual title, optionally with the title that RCT2 gives it in brackets.

```suggestion
                "name": "Die Regimentskinder (Children of the regiment)",
```

File: objects/rct2/music/rct2.music.fairground.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                "composer": "Jonny Heykens"
+            },
+            {
+                "source": "$RCT2:DATA/css6.dat",
+                "name": "In Continental Mood",
+                "composer": "Unknown"
+            },
+            {
+                "source": "$RCT2:DATA/css7.dat",
+                "name": "Wedding Journey",
```
Comment (Gymnasiast) [inline objects/rct2/music/rct2.music.fairground.json:28]:
```suggestion
                "name": "Traumfahrt (Wedding Journey)",
```

File: objects/rct2/music/rct2.music.fairground.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                "composer": "Traumfahrt; traditional"
```
Comment (Gymnasiast) [inline objects/rct2/music/rct2.music.fairground.json:29]:
```suggestion
                "composer": "Traditional"
```

File: objects/rct2/music/rct2.music.fairground.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    "originalId": "0000008E|#MUSIC01|00000000",
+    "sourceGame": ["rct2"],
+    "objectType": "music",
+    "properties": {
+        "rideTypes": ["merry_go_round"],
+        "tracks": [
+            {
+                "source": "$RCT2:DATA/css4.dat",
+                "name": "Children of the regiment",
+                "composer": "Die Regimentskinder; Julius Fučík"
```
Comment (Gymnasiast) [inline objects/rct2/music/rct2.music.fairground.json:14]:
```suggestion
                "composer": "Julius Fučík"
```

File: objects/rct2/music/rct2.music.pirate.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    "version": "1.0",
+    "originalId": "0000008E|#MUSIC30|00000000",
+    "sourceGame": ["rct2"],
+    "objectType": "music",
+    "properties": {
+        "originalStyleId": 30,
+        "tracks": [
+            {
+                "source": "$RCT2:DATA/css44.dat",
+                "name": "Drunken Sailor",
```
Comment (Gymnasiast) [inline objects/rct2/music/rct2.music.pirate.json:13]: I was going to comment on this, but Wikipedia appears to favour "Drunken Sailor".

**Comment by Gymnasiast** [comment]
Looks good in general. What is your plan with the circus style? I can't seem to find it in here - do you plan to make it behave more like a proper sound effect?

**Comment by IntelOrca** [comment]
> Looks good in general. What is your plan with the circus style? I can't seem to find it in here - do you plan to make it behave more like a proper sound effect?

Based on the fact you can not select or change the music track for Circus, I do not consider it ride music.

> Edit: and how about the title music?

I am leaving that for now as you can not currently use it as ride music anyway. We can add it as an object later if need be.

> Edit 2: sourceGame would have to be updated to also include RCT1 where appropriate. However, it doesn't have to be in this PR - I can add it later on.

I'll leave that to you then. 😉

**Comment by Gymnasiast** [comment]
> Based on the fact you can not select or change the music track for Circus, I do not consider it ride music.

That's correct, but I was mostly talking from a technical standpoint - how will the game handle it? It is currently still technically ride music, even though it behaves like a sound effect.

**Pull request merged**